Jenkinsfile

```diff
@@ -28,7 +30,7 @@ pipeline {
       }
       steps {
         withDockerRegistry(url: "https://index.docker.io/v1/", credentialsId: 'dockerbuildbot-index.docker.io') {
-          sh 'make push'
+          sh 'export DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE=$params.PASSPHRASE && make push'
```

Comment:
a more clean way of structuring this line could be 

```
make DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE=$params.PASSPHRASE push
```

this is fine with me though

Reply:
sorry, I accidentally closed and deleted this, but I'll make the change in a new PR I opened here:

https://github.com/docker/binfmt/pull/10

Thank you for the feedback!!